packages/gen-ai-hub/src/orchestration/orchestration-client.test.ts

```diff
@@ -28,92 +37,89 @@ describe('GenAiHubClient', () => {
     nock.cleanAll();
   });
 
-  it(' calls chatCompletion and parses response', async () => {
+  afterAll(() => {
+    clearXsuaaServices();
+  });
```

Comment:
[q] Why is this required? The test seems to pass, even if we skip this.

Reply:
It's a left over from testing with the real endpoint. I'll remove this.

Reply:
More tests to be added in a follow-up PR.